src/openforms/js/components/admin/submissions/filter.js

```diff
@@ -0,0 +1,47 @@
+const init = async () => {
+  // Retrieve the filter link that displays only triggered rules
+  const triggeredLink = document.querySelector('.logic-rules-filter > .selected > a');
+
+  // Only triggered rules are displayed
+  document
+    .querySelectorAll('.logic-logs__col-body--trigger-' + triggeredLink.dataset.trigger)
+    .forEach(trigger => {
+      trigger.parentNode.classList.remove('logic-logs__row-body--hidden');
+    });
+};
+
+document.querySelectorAll('.logic-rules-filter__link').forEach(link => {
+  link.addEventListener('click', event => {
+    event.preventDefault();
+
+    // Hiding all the rows first
+    document.querySelectorAll('.logic-logs__row-body').forEach(row => {
+      row.classList.add('logic-logs__row-body--hidden');
+    });
+
+    const filterValue = event.target.dataset.trigger;
+
+    // Change the focus of the selected filter
+    document.querySelector('.selected').classList.remove('selected');
```

Comment:
I'd make this queryselector a bit more specific to prevent unintended side-effects elsewhere:

```suggestion
    document.querySelector('.logic-rules-filter .selected').classList.remove('selected');
```